apps/web/components/blog/blog-post-page.tsx

```
@@ -0,0 +1,294 @@
+          <div className="max-w-4xl mx-auto">
+            <div className="grid grid-cols-1 lg:grid-cols-4 gap-12">
+              {/* Main Content */}
+              <motion.article
+                initial={{ opacity: 0, y: 20 }}
+                animate={{ opacity: 1, y: 0 }}
+                transition={{ duration: 0.8, delay: 0.2 }}
+                className="lg:col-span-3"
+              >
+                <div 
+                  className="prose prose-invert prose-purple max-w-none prose-lg"
+                  dangerouslySetInnerHTML={{ __html: mockPost.content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)